py4cast/datasets/poesy.py

```diff
@@ -309,6 +384,7 @@ def __init__(
         self.shuffle = self.split == "train"
         self.cache_dir.mkdir(parents=True, exist_ok=True)
         self.step_duration = self.settings.term["timestep"]
+        self.browse_dataset = browse_poesy
```

Comment:
do we need to add this as an attribute ? it is defined in the same file.

Reply:
I thought it would be more clear to create an attribute rather to use a function defined in the same file. (If we want to define several functions to navigate into the dataset, it could be useful for instance.)